web/src/utils/helpers.ts

```diff
@@ -1,4 +1,5 @@
 import { callerLocation, useMeta } from 'api/getMeta';
+import { memoize } from 'es-toolkit';
```

Comment:
Lol, didn't even know that there's a new `lodash` in town 😄 
Seems pretty overkill for just this memo function IMO:

![Screenshot 2025-02-18 at 21 37 27](https://github.com/user-attachments/assets/453aa275-a129-46a7-9306-d13983615060)

That said I'm not against having it per se, but I do feel like it might be more trouble than what it's worth at this point :)

Reply:
That's the benefit of the ES-toolkit, fully modular and uses es modules so if we are only using the memorize function that's the only code we are importing!

So size wise the addition is pretty small. About as much code as the basic boilerplate for one component.

Reply:
Nice, there's probably also other custom code we could get rid of then 🙂